src/components/draggable-list.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState,useRef } from 'react';
+import { Button, Tile } from 'carbon-components-react';
+import { css, cx } from 'emotion';
+import {
+	AddAlt32,
+	Draggable16,
+	TrashCan32
+} from '@carbon/icons-react';
+
+const addDragTargetStyle = css`
+	height: var(--drag-target-height, 32px);
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	outline-offset: -2px;
+	transition: height 0.15s ease-in-out;
+`;
+
+const addButtonDraggingStyle = css`
+	outline: var(--outline, 1px dashed #1666fe);
+`;
+
+const addButtonStyle = css`
+	width: 100%;
+	display: flex;
+	justify-content: center;
+`;
+
+const trashButtonStyle = css`
+	margin-left: 12px;
+	top: 0;
+	right: 0;
+	border-color: transparent;
+
+	&.bx--btn.bx--btn--icon-only.bx--tooltip__trigger {
+		position: absolute;
+	}
+`;
+
+const draggableIconStyle = css`
+	position: absolute;
+	top: 50%;
+	left: 0;
+`;
+
+const tileStyle = css`
+	position: relative;
+`;
+
+export const DraggableTileList = ({
+	template, // Functional component
+	dragOver = (_: any) => { },	// Override onDragOver event
```

Comment:
```suggestion
	dragOver = (_: any) => false,	// Override onDragOver event
```
?

Reply:
Why false by default? That would make it so that component nothing can be dropped in the drop zone and would require a mandatory dragOver function to be passed. I think it makes more sense to change it to:
```suggestion
	dragOver = (_: any) => true,	// Override onDragOver event
```